doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
-on that compute host.
+Step 1: Configure
+'''''''''''''''''
+You can specify the run settings of a model.
+In this experiment, we invoke the Python interpreter to run
+the python script defined in section: The Application Script.
+To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.
+The function returns a ``RunSettings`` object.
+When initializing the ``RunSettings`` object,
+we specify the path to the application file,
+`application_script.py`, for
+``exe_args``, and the run command for ``exe``.
+.. code-block:: python
+
+  # Initialize a RunSettings object
+  model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/standard_orch_model.py")
+  model_settings.set_nodes(1)
+
+Step 2: Initialize
+''''''''''''''''''
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``:
+.. code-block:: python
+
+  # Initialize the Model
+  model = exp.create_model("model", model_settings)
+
+Step 3: Start
+'''''''''''''
+Next, launch the model instance using the ``Experiment.start()`` function.
+.. code-block:: python
+
+  # Launch the Model
+  exp.start(model, block=True, summary=True)
+
+.. note::
+    We specify `block=True` to ``exp.start()`` because our experiment
+    requires that the ``Model`` finish before the experiment continues.
+    This is because we will request tensors from the database that
+    are inputted by the Model we launched.
+
+Poll Data Using Clients
+-----------------------
+Next, check if the tensor exist in the standard database using ``Client.poll_tensor()``.
+This function queries for data in the database. The function requires the tensor name (`name`),
+how many milliseconds to wait in between queries (`poll_frequency_ms`),
+and the total number of times to query (`num_tries`):
+.. code-block:: python
+
+  # Retrieve the tensors placed by the Model
+  value_2 = driver_client_standard_db.poll_key("tensor_2", 100, 100)
+  # Validate that the tensor exists
+  logger.info(f"The tensor is {value_2}")
+
+The output will be as follows::
+
+  test
+
+Cleanup Experiment
+------------------
+Finally, use the ``Experiment.stop()`` function to stop the database instances. Print the
+workflow summary with ``Experiment.summary()``:
+
+.. code-block:: python
+
+  # Cleanup the database
+  exp.stop(standard_db)
+  logger.info(exp.summary())
+
+When you run the experiment, the following output will appear::
+ test
+
+====================
+Colocated Deployment
+====================
+--------
+Overview
+--------
+During colocated deployment, a SmartSim ``Orchestrator`` (the database) is launched on
+the ``Model`` compute node(s).
+The orchestrator is non-clustered and each ``Model`` compute node hosts an instance of the database.
+Processes on the compute host individually address the database.
```

Comment:
We seemed to talk a lot in the clustered deployment section about what it's good for (scaling, primarily). Can we replicate that idea here and say that a colo db is good for ____ with the tradeoffs of losing the ability to scale and sharing information with other processes?